spec/trace/exporter/otlp_spec.lua

```diff
@@ -36,6 +36,8 @@ describe("encode_spans", function()
         local resource = encoded.resource_spans[1]
         assert(#resource.instrumentation_library_spans == 1)
         assert(#resource.instrumentation_library_spans[1].spans == 10)
+        assert(resource.instrumentation_library_spans[1].spans[1].start_time_unix_nano == "123456788")
+        assert(resource.instrumentation_library_spans[1].spans[1].end_time_unix_nano == "123456789")
```

Comment:
Just a nice addition here to check that we are setting and exporting the times we expect on spans